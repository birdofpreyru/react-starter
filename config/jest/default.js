const config = require('topcoder-react-utils/config/jest/default');

module.exports = config;
