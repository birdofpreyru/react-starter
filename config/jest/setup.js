import '@dr.pogodin/react-utils/config/jest/setup';
