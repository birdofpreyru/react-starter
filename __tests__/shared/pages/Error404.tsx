/** @jest-environment jsdom */

import { GlobalStateProvider } from '@dr.pogodin/react-utils';

import { snapshot } from '@dr.pogodin/react-utils/jest';

import Error404Page from 'pages/Error404';

test('Matches shallow snapshot', () => {
  snapshot(
    <GlobalStateProvider initialState={{}}>
      <Error404Page />
    </GlobalStateProvider>,
  );
});
