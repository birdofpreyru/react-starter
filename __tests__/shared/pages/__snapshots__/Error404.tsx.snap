// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<div
  class="T3cuHB"
>
  <div
    class="lqNh4h "
  />
  <div
    class="pPlQO2"
  >
    <h1>
      Error 404: Requested resource is not found.
    </h1>
  </div>
  <div
    class="lqNh4h "
  />
</div>
`;
